gc/base/Configuration.cpp

```diff
@@ -455,6 +456,49 @@ MM_Configuration::initializeGCThreadCount(MM_EnvironmentBase* env)
 	}
 }
 
+void
+MM_Configuration::initializeGCParameters(MM_EnvironmentBase* env)
+{
+	MM_GCExtensionsBase* extensions = env->getExtensions();
+	
+	/* TODO 108399: May need to adjust -Xmn*, -Xmo* values here if not fully specified on startup options */
+
+	/* initialize packet lock splitting factor */
+	if (0 == extensions->packetListSplit) {
+		if (16 >= extensions->gcThreadCount) {
+			extensions->packetListSplit = extensions->gcThreadCount;
+		} else if (32 >= extensions->gcThreadCount) {
+			extensions->packetListSplit = 16 + ((extensions->gcThreadCount - 16) / 4);
+		} else {
+			extensions->packetListSplit = 20 + ((extensions->gcThreadCount - 32) / 8);
+		}
+	}
+
+	/* initialize scan cache lock splitting factor */
+	if (0 == extensions->cacheListSplit) {
```

Comment:
this looks like scavenger specific code which should be under the compile time flag